ctapipe_io_magic/__init__.py

```diff
@@ -92,6 +94,68 @@
     DATA_MAGIC_LST_TRIGGER: EventType.SUBARRAY,
 }
 
+class ReportLaserContainer(Container):
+    """ Container for Magic laser parameters """
+    UniqueID = Field(List[Any], 'No.')
+    Bits = Field(List[Any], 'ID')
+    MJD = Field(List[Any], 'Modified Julian Date')
+    BadReport = Field(List[Any], 'Bad Report') # to be improved
+    State = Field(List[Any], 'State') # to be	improved
+    IsOffsetCorrection = Field(List[Any], 'Is Offset Correction')
+    IsOffsetFitted = Field(List[Any], 'Is Offset Fitted')
+    IsBGCorrection = Field(List[Any], 'Is BG Correction')
+    IsT0ShiftFitted = Field(List[Any], 'Is T0 Shift Fitted')
+    IsUseGDAS = Field(List[Any], 'Is Use GDAS')
+    IsUpwardMoving = Field(List[Any], 'Is Upward Moving')
+    OverShoot = Field(List[Any], 'Over Shoot')
```

Comment:
why not np.int for fields with type Int_t (overshoot, undershoot, ...)

Reply:
Now we use Any everywhere